Parsers/ASimAuditEvent/Parsers/ASimAuditEventCiscoISE.yaml

```diff
@@ -0,0 +1,76 @@
+    let ASIM_GetUsernameType = (username:string) { 
+      case ( 
+          username contains "@" , "UPN"
+          , username contains "\\", "Windows"
+          , isempty(username), ""
+          , "Simple"
+      )
+    };
+    let CiscoISEAuditParser=(disabled:bool=false){
+      Syslog
+      | where not(disabled)
+      | where ProcessName has_any ("CISE", "CSCO")
+      | extend EventOriginalType = extract(@".*\d{10}\s(\d{4,5})\s.*", 1, SyslogMessage)
```

Comment:
Value of EventOriginalType and EventOriginalSeverity can be extracted using parse statement as the initial format of the event is the same. This would avoid the use of inefficient regex experessions.